java/ql/src/semmle/code/java/dataflow/internal/TaintTrackingUtil.qll

```diff
@@ -263,6 +264,8 @@ private predicate taintPreservingQualifierToMethod(Method m) {
   )
   or
   m.(TaintPreservingCallable).returnsTaintFrom(-1)
+  or
+  exists(JaxRsResourceMethod resourceMethod | m.(GetterMethod).getDeclaringType() = resourceMethod.getAParameter().getType())
```

Comment:
@smowton I saw your [PR for improving JAX-WS models](https://github.com/github/codeql/pull/5491/), but couldn't find taint propagation for a getter called on a `JaxRsResourceMethod` parameter.

I added this to avoid bringing https://github.com/github/codeql/blob/ef0ea247c40da805efa427a37f5213457d18f714/java/ql/src/experimental/Security/CWE/CWE-094/FlowUtils.qll from experimental (since the query was using `hasGetterFlow`), but let me know if there's a better way of doing this.